packages/amplify-util-uibuilder/package.json

```diff
@@ -16,8 +16,8 @@
   "dependencies": {
     "@aws-amplify/amplify-cli-core": "4.0.5",
     "@aws-amplify/amplify-prompts": "2.7.0",
-    "@aws-amplify/codegen-ui": "2.13.1",
-    "@aws-amplify/codegen-ui-react": "2.13.1",
+    "@aws-amplify/codegen-ui": "2.13.2-ad9ba60.0",
```

Comment:
this version will eventually make it's way into the main CLI release. Do we intend to be depending on a prerelease version of codegen-ui?

Reply:
I did this for the sake of time, but the new version is out now. Ill rev.